tools/setup/emoji/build_emoji

```diff
@@ -240,6 +240,62 @@ def setup_emoji_farm(cache_path, emoji_map, emoji_data):
 
     generate_sprite_css_files(cache_path, emoji_map, emoji_data)
 
+def setup_old_emoji_farm(cache_path, emoji_map):
+    # type: (Text, Dict[Text, Text]) -> None
+    # Code for setting up old emoji farm.
+    remapped_emojis = {
+        "0023": "0023-20e3",    # Hash
+        "0030": "0030-20e3",    # Zero
+        "0031": "0031-20e3",    # One
+        "0032": "0032-20e3",    # Two
+        "0033": "0033-20e3",    # Three
+        "0034": "0034-20e3",    # Four
+        "0035": "0035-20e3",    # Five
+        "0036": "0036-20e3",    # Six
+        "0037": "0037-20e3",    # Seven
+        "0038": "0038-20e3",    # Eight
+        "0039": "0039-20e3",    # Nine
+        "1f48f": "1f469-200d-2764-200d-1f48b-200d-1f468",    # Couple kiss
+        "1f491": "1f469-200d-2764-200d-1f468",  # Couple with heart
+    }
+
+    os.chdir(cache_path)
+    cache_emoji = os.path.join(cache_path, 'images', 'emoji')
+    cache_emoji_unicode = os.path.join(cache_path, 'images', 'emoji', 'unicode')
+    cache_google_64 = os.path.join(cache_path, 'images-google-64')
+
+    # Symlink zulip.png image file as well as check for symlink support.
+    try:
+        image_file_path = os.path.join(cache_google_64, 'zulip.png')
+        symlink_path = os.path.join(cache_emoji, 'zulip.png')
+        os.symlink(image_file_path, symlink_path)
+
+        symlink_path = os.path.join(cache_emoji_unicode, 'zulip.png')
+        os.symlink(image_file_path, symlink_path)
+    except OSError:
+        raise Exception("Error: Unable to create symlinks. Make sure you have permission to create symbolic links.")
```

Comment:
I think it might be a good idea to just do a `try/except` for symlink creation inside a tempdir inside `provision` directly, rather than here, since it's kinda weird to just do it in the first thing that uses a symlink as we currently do.  We can add such a check as a follow-up commit.

Reply:
Also, once we make the symlink issue, we could just do `emoji_map['zulip'] = 'zulip'` and have the below handle this case, right?

Reply:
Just checked the provision script and found that we already do such a check there so removed this and ab another one.

>Also, once we make the symlink issue, we could just do emoji_map['zulip'] = 'zulip' and have the below handle this case, right?

Not until emoji web PR gets merged, since for now we are using emoji_map for other purposes as well which assume that it will not contain any realm emoji.